changelog.go

```diff
@@ -47,6 +51,9 @@ func (cl *ChangeLog) filterReleases(rels []*github.RepositoryRelease) []*github.
 // Generate generates changelog text from given releases and outputs it to its writer
 func (cl *ChangeLog) Generate(rels []*github.RepositoryRelease) error {
 	rels = cl.filterReleases(rels)
+	if len(rels) == 0 {
+		fail(fmt.Errorf("no releases to process, all filtered out"))
+	}
```

Comment:
I don't think this is an error. No release is fine to generate a first empty changelog.

Reply:
Ah right! Just thought to align it with this:
https://github.com/rhysd/changelog-from-release/blob/3cbaf65deff192c56a7ab759a630431b2aa9d63e/main.go#L83-L85

But it's probably better to leave it as it is. I'm going to adjust it! 👍

Reply:
Ah, I see. I need to remove the check after merging this PR.

Reply:
Great, thanks for your prompt review! :smiley: